### PR TITLE
Initial upload of Unit Test Framework

### PR DESCRIPTION
**Pull request by ranveersandhuamd**
Initial upload of Unit Test Framework

Signed-off-by: Ranveer Sandhu <ranveer.sandhu@amd.com>

### CODE REVIEW

File: AmdCommonPkg/Test/UnitTest/AgesaModuleUtPkg.dsc.inc
Comment (pbgrimes) [inline AmdCommonPkg/Test/UnitTest/AgesaModuleUtPkg.dsc.inc:1]: Need to fix line endings + remove special chars

**Review by pbgrimes — COMMENTED** [review]
Need to apply MIT license header to all files.

File: AmdCommonPkg/Test/UnitTest/AgesaModuleUtPkg.dsc.inc
Comment (ranveersandhuamd) [reply AmdCommonPkg/Test/UnitTest/AgesaModuleUtPkg.dsc.inc:1]: Done

**Comment by pbgrimes** [comment]
*MIT headers not needed here (Beta License applies to entire repository).  Can add MIT headers if code is moved to a separate repo.

**Review by pbgrimes — APPROVED** [review]

**Pull request merged**